src/app/item-popup/ItemSocketsGeneral.tsx

```diff
@@ -86,8 +87,13 @@ export default function ItemSocketsGeneral({ item, minimal, onPlugClicked }: Pro
             >
               {!minimal && (
                 <div className={styles.exoticDescription}>
-                  <RichDestinyText
-                    text={exoticArmorPerkSocket.plugged.plugDef.displayProperties.description}
+                  <ClarityDescriptionConstructor
+                    hash={exoticArmorPerkSocket.plugged.plugDef.hash}
+                    fallback={
+                      <RichDestinyText
+                        text={exoticArmorPerkSocket.plugged.plugDef.displayProperties.description}
+                      />
+                    }
```

Comment:
I think we probably want to add the community description on top, rather than replacing it - the description from the manifest is localized so I'd hate to lose it.

Reply:
Some description are pretty big both can be messy
Maybe option to display both or only one of them
Or make them collapsible

Reply:
Since these appear in hover tooltips, I'm not sure we can have clickable elements in there

Reply:
Added option to choose how to display them instead